debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/J9IndexableObjectHelper.java

```diff
@@ -291,8 +291,11 @@ public static void getData(J9IndexableObjectPointer objPointer, Object dst, int
 			getBooleanData(objPointer, (boolean[])dst, start, length, destStart);
 			break;
 		}
-			
-		case 'L':
+
+		/*[IF INLINE-TYPES]*/
+		case 'Q': /* fall through */
+		/*[ENDIF] INLINE-TYPES*/
```

Comment:
here and below: do we really need this commented out `[IF INLINE-TYPES]` ?

Reply:
Having preprocessor directives commented is the convention used throughout the codebase (I'm assuming so that the java code can still be built without the preprocessor).

Reply:
They need to start with `/*[` since that is that pattern the preprocessor looks for. 

We don't use the code un-preprocessed any more, although that may have been the intention long ago. We do want as much un-preprocessed code uncommented as possible, and to compile, so search tools recognize and find it.

Reply:
Ah, thanks for the clarification Peter!